benchopt/plotting/html/static/result.js

```diff
@@ -651,6 +666,27 @@ const createLegendItem = (solver, color, symbolNumber) => {
 }
 
 
+function createSolverDescription(legendItem, {title, description}) {
+  // skip if no description was provided
+  if (description === null || description === undefined || description === "")
+    return legendItem;
```

Comment:
I think I would prefer to have a legend but with `no description provided` in this case, otherwise, this is weird that there is no hover effect in some cases but a popup in others.

Reply:
```suggestion
  if (description === null || description === undefined || description === "")
    description = "No description provided";
```